projects/cuelang.org/package.yml

```diff
@@ -0,0 +1,31 @@
+distributable:
+  url: https://github.com/cue-lang/cue/archive/refs/tags/v{{version}}.tar.gz
+  strip-components: 1
+
+versions:
+  github: cue-lang/cue/tags
+
+provides:
+  - bin/cue
+
+build:
+  script: |
+    go mod download
+    go build -v -ldflags="$LDFLAGS" -o "{{ prefix }}"/bin/cue ./cmd/cue
+  dependencies:
+    go.dev: ^1.18
+  env:
+    LDFLAGS:
+      - -s
+      - -w
+      - -X cuelang.org/go/cmd/cue/cmd.version=v{{ version }}
+    linux:
+      # or segmentation fault
+      # fix found here https://github.com/docker-library/golang/issues/402#issuecomment-982204575
+      LDFLAGS:
+        - -buildmode=pie
+
+test:
+  script: |
+    out=$(cue export data.cue --out yaml)
+    test "$out" = "project: cuelang.org"
```

Comment:
This is slightly more than the bare minimum (e.g. `cue version`). Happy to add more test if you feel it's appropriate.